buyer/cancel.md

```diff
@@ -2,21 +2,89 @@
 description: How to request the supplier to cancel an order or line
 ---
 
-# Cancel an order
+# Cancelling an order or line
 
-If a purchase order line is `Issued` , `In Progress`, `Confirmed`or `Rejected` the buyer may `cancel` the line immediately.
+Cancel an order line in Tradecloud when it is cancelled at the buyer.
 
-The buyer can cancel by setting `indicators.cancel`on either order or line level and update the order.
+- `Issued`, `In Progress`, `Rejected` and `Confirmed` lines will become `Cancelled` immediately (without request)
+- `Completed` lines cannot be cancelled
+- `Cancelled` lines cannot be cancelled again
 
-{% page-ref page="update.md" %}
+## Cancelling by resending an order
+
+The order or line can be cancelled by setting `indicators.cancelled` on either order or line level and updating the order.
 
-{% hint style="warning" %}
-You cannot cancel a`Cancelled` or`Completed` line
+{% hint style="info" %}
+If you provide a `cancelled` indicator on order level, **ONLY** the lines provided in this order message will be cancelled.
+
+If you also provide a `cancelled` indicator on line level, it has **precedence** over the order level `cancelled` indicator.
 {% endhint %}
 
+{% page-ref page="update.md" %}
+
+## Cancelling by sending the cancelled indicator
+
+The order or can be cancelled by setting `indicators.cancelled` on either order or line level and sending this indicator only.
+
 {% hint style="info" %}
-Order lines having process status `Issued, In Progress, Confirmed` or `Rejected` will become `Cancelled`
+If you provide a `cancelled` indicator on order level, **ALL** the lines in the order will be cancelled.
+
+If you also provide a compl`cancelled`eted indicator on line level, it has **precedence**  over the order level `cancelled` indicator.
```

Comment:
Typo `compl'cancelled'eted`

Reply:
I would also add `cancelld or any other indiciator` because if you send line `completed` indicator it will override order `cancelled` indicator

Reply:
Typo fixed, nice copy-pasta

Reply:
@oleg has `completed` also override `delivered` indicator? In practice the buyer can send both I know from legacy experience.

Reply:
`completed` and `delivered` can be specified together. I would say they can complement each other
@Sjorstradecloud Please confirm

Reply:
Yes `completed` and `delivered` indicators can, and will be, used together.